src/main/resources/db/changelog/changes/09-create-books_categories-table.yaml

```diff
@@ -0,0 +1,26 @@
+databaseChangeLog:
+  - changeSet:
+      id: create_books_categories_table
+      author: shastkiv
+      changes:
+        - createTable:
+            tableName: books_categories
+            columns:
+              - column:
+                  name: book_id
```

Comment:
Id is also unique, I assume that there is some PK constraints that includes all necessary constraints please check it